tests/sentry/metrics/test_minimetrics.py

```diff
@@ -254,6 +267,7 @@ def __getattr__(self, name):
 @override_options(
     {
         "delightful_metrics.minimetrics_sample_rate": 1.0,
+        "delightful_metrics.enable_span_attributes": False,
```

Comment:
Just a question, do we have to define this with `False` value if also default value of the flag is `False`?

Reply:
yeah, tests start failing if it is not there :D